test/data/v3/with-k8s-fake-client/tests/lookup_test.yaml

```diff
@@ -0,0 +1,20 @@
+templates:
```

Comment:
possibly more example needed.

`kubernetes_provider` in top level is missing as well

Reply:
i added more examples.

What do you mean kubernetes_provider is missing at the top level?

Reply:
example. Top level config e.g. suite level.

```yml
templates:
  - templates/lookup.yaml
 kubernetesProvider:
      scheme:
        "v1/Namespace":
          gvr:
            version: "v1"
            resource: "namespaces"
          namespaced: false
        "v1/Pod":
          gvr:
            version: "v1"
            resource: "pods"
          namespaced: true
        "networking.k8s.io/v1/Ingress":
          gvr:
            group: "networking.k8s.io"
            version: "v1"
            resource: "ingresses"
          namespaced: true
tests:
  - it: manifest should match snapshot
    asserts:     
      - equal:
          path: pod_exists.metadata.name
          value: exists
      - equal:
          path: pod_exists.metadata.namespace
          value: default
      - lengthEqual:
          path: namespaces.items
          count: 2
      objects:
        - kind: Pod
          apiVersion: v1
          metadata:
            name: exists
            namespace: default
```

Reply:
probably worth to consider something like ^ for a future release